app/build.gradle

```diff
@@ -32,8 +32,8 @@ android {
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
         testInstrumentationRunnerArguments clearPackageData: 'true'
         manifestPlaceholders.isRaptorEnabled = "false"
-        buildConfigField "boolean", "IS_RELEASED", "false"
-        resValue "bool", "IS_NOT_RELEASED", "true"
+        buildConfigField "String", "CHANNEL", '"' + "debug" + '"'
```

Comment:
Do we really need a new BuildConfig channel field when the product field carries the same information?